src/frontend.rs

```diff
@@ -724,6 +727,9 @@ impl AsyncComponent for App {
                 glib::Propagation::Stop
             }
         });
+
+        let menu_event = MenuEvent::receiver();
```

Comment:
Construct it closer to the place where it is actually used, probably before `sender.command`, otherwise you may create a receiver when there is no tray icon at all and it is harder to read when things are so far apart.